manifests/charts/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -124,6 +124,9 @@ spec:
         add:
         - NET_ADMIN
         - NET_RAW
+    {{- else if eq (annotation .ObjectMeta `sidecar.istio.io/interceptionMode` .ProxyConfig.InterceptionMode) `TPROXY` }}
+        add:
+        - NET_ADMIN
```

Comment:
The istio-validator container needs NET_ADMIN in order to work with TPROXY. This didn't cause any issues previously, since the validation traffic was localhost-to-localhost, which does not require IP_TRANSPARENT to be set. Now, the traffic is localhost-to-podIP, so this is needed.

The istio-proxy container already runs with NET_ADMIN for TPROXY mode, so this change simply makes the istio-validator more accurate in its validation.